linux/net/rina/pdu.c

```diff
@@ -287,3 +288,9 @@ int pdu_destroy(struct pdu * p)
         return 0;
 }
 EXPORT_SYMBOL(pdu_destroy);
+
+uint_t pdu_length(struct pdu * pdu) {
+	return buffer_length(pdu->buffer) +
+		serdes_pci_size(pci_type(pdu->pci));
+}
+EXPORT_SYMBOL(pdu_length);
```

Comment:
isnt't this length actually the serialized length? If so, shouldn't it be in serdes.c or pdu-ser.c?